frontend/components/Vehicles/VehicleDialog.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <v-dialog v-model="showDialog" max-width="500px">
+    <template v-slot:activator="{ on, attrs }">
```

Comment:
Consider using shorthand for a v-slot
```suggestion
    <template #activator="{ on, attrs }">
```